topydo/commands/ListCommand.py

```diff
@@ -88,11 +88,10 @@ def _print(self):
             # create a standard printer with some filters
             indent = config().list_indent()
             hidden_tags = config().hidden_tags()
+            format_parser = ListFormatParser(self.todolist, self.format)
 
             filters = []
-            filters.append(PrettyPrinterFormatFilter(self.todolist,
-                           self.format))
-            filters.append(PrettyPrinterAlignFilter())
+            filters.append(PrettyPrinterFormatFilter(self.todolist, format_parser))
```

Comment:
I would like to see that `PrettyPrinterFormatFilter` instantiates the parser, rather than putting that responsibility to the client of the filter.